### ISSUE TITLE
Issue getting Hubitat to show in "Integrations"

### ISSUE THREAD

**Issue by cknic** [jason0x43/hacs-hubitat#110]
I have installed this integration using the HACS method, however despite restarting the service and even the server itself multiple times I can't get the "Hubitat" to show in the "Configuration" -> "Integrations" list at all.  I'm not seeing any real log errors either.

Any ideas or areas I should dig into?

**Comment by cknic** [comment]
of course because i opened this i did a fourth reboot and it worked.... idk

**Comment by jason0x43** [comment]
This issue has come up several times, and I have yet to find the cause. Tricky.

**Comment by uberjay** [comment]
I ran into the same problem here with a manual installation (copying the hubitat directory into `custom_components`) -- I was about to give up, and hit shift-reload in desperation.... and there it was, in the integrations list!

My home assistant installation is of the virtual machine "Home Assistant OS" variety. (version 2021.5.5) I tested both with a fresh checkout of the `master` branch of the hubitat integration, as well as v0.6.8.
